ortho_seq_code/gui/job_runner.py

```diff
@@ -17,9 +17,9 @@ def job_func(self, progress_callback):
             self.molecule,
             self.poly_order,
             self.precomputed,
-            out_dir=".",
+            out_dir="../results_ortho_seq_testing/",
```

Comment:
How come this has a hard coded output path?

Reply:
We currently don't have an out_dir button for the gui, so for now this works. But definitely need to have an out_dir button where the user can select the directory where they want their output results to go.